protocol/xmrmaker/event.go

```diff
@@ -251,12 +251,7 @@ func (s *swapState) handleEvent(event Event) {
 }
 
 func (s *swapState) handleEventETHLocked(e *EventETHLocked) error {
-	resp, err := s.handleNotifyETHLocked(e.message)
-	if err != nil {
-		return err
-	}
-
-	return s.SendSwapMessage(resp, s.ID())
+	return s.handleNotifyETHLocked(e.message)
```

Comment:
Should the caller just invoke `handleNotifyETHLocked` directly? I'm looking at the calling code, and it seems counterintuitive that the error channel is passed here, but any returned error is only pushed on to the channel by the calling code.

Reply:
updated